timelock-impl/src/main/java/com/palantir/atlasdb/timelock/management/PersistentNamespaceLoader.java

```diff
@@ -22,6 +22,7 @@
 public interface PersistentNamespaceLoader {
     /**
      * Gets all namespaces that have been persisted (via the persistence method under question).
+     * No transactionality guarantees are given.
```

Comment:
Tbh, I don't think this is strictly needed. If we want this endpoint to be used in just the one place we are thinking of, then I can't see how anyone would expect any different